tests/snippets/builtin_locals.py

```diff
@@ -0,0 +1,8 @@
+
+a = 5
+b = 6
+
+loc = locals()
+
+assert loc['a'] == 5
```

Comment:
(Could you do a similar test inside a function? This would help in case we try optimizing locals access in the future.)

Reply:
Sure :grinning: